visivo/commands/utils.py

```diff
@@ -9,35 +9,28 @@
 from visivo.utils import load_yaml_file
 
 
-def find_or_create_target(project: Project, target_or_name: str) -> Target:
-    if len(project.targets) == 0 and not target_or_name:
+def find_default_target(project: Project, target_name: str) -> Target:
+    if len(project.targets) == 0 and not target_name:
         raise click.ClickException(
-            f"The project must contain a target or a target must be passed as an object on the command line."
+            f"The project must contain a target."
```

Comment:
I wonder if it might be helpful to link to the target in the docs. I think our URLs are pretty stable at this point: https://docs.visivo.io/reference/configuration/Target/

Reply:
The only reason I wouldn't want to do that is if stuff moves then we have to go back and change it all.